tests/integration/inventory.networking

```diff
@@ -1,8 +1,8 @@
 [aci]
-cn-dmz-apic-m1-02-v42 ansible_host=173.36.219.68 aci_hostname=173.36.219.68
-cn-dmz-apic-m1-03-v52 ansible_host=173.36.219.69 aci_hostname=173.36.219.69
+#cn-dmz-apic-m1-02-v42 ansible_host=173.36.219.68 aci_hostname=173.36.219.68
```

Comment:
Should these hosts be commented out?

Reply:
yes, it should. I just forgot to remove then.